tests/sentry/seer/test_math.py

```diff
@@ -0,0 +1,54 @@
+import math
+
+from sentry.seer.math import entropy, kl_divergence, laplace_smooth, relative_entropy, rrf_score
+
+
+def test_laplace_smooth():
+    assert laplace_smooth([0.5, 0.5]) == [0.5, 0.5]
+    assert laplace_smooth([100, 100]) == [0.5, 0.5]
+
+    result = laplace_smooth([75, 25])
+    assert math.isclose(result[0], 0.75, rel_tol=1e-3)
+    assert math.isclose(result[1], 0.25, rel_tol=1e-3)
+
+    result = laplace_smooth([67, 33])
+    assert math.isclose(result[0], 0.67, rel_tol=1e-3)
+    assert math.isclose(result[1], 0.33, rel_tol=1e-3)
```

Comment:
should we add some test to make sure that it actually fixes zeros?

```suggestion
    assert math.isclose(result[1], 0.33, rel_tol=1e-3)
    
    result = laplace_smooth([0, 1])
    assert result[0] > 0
```